fab/fab/operations/supervisor.py

```diff
@@ -97,7 +98,7 @@ def set_celery_supervisorconf():
         num_workers = params.get('num_workers', 1)
 
         params.update({
-            'queue': queue,
+            'comma_separated_queue_names': comma_separated_queue_names,
```

Comment:
Should `celery_periodic` and `flower` be removed from this list before adding it, or maybe raise an exception if those appear grouped with others?

Reply:
I think it's actually fine for `celery_periodic` to be grouped with the others? (Its presents will also make celery _beat_ get auto-created, but I think it's fine for the queue itself to be processed in the same process as other queues.) Maybe I'm wrong; that was my understanding.

`flower` though, yes, since that's not really a queue at all. It feels like the right time to check this is when checking in code changes, not when fab deploying, so I'm thinking of writing some validation for that file and putting it in a travis test, rather than making it fail here. Thoughts?

Reply:
I implemented the travis test I was suggesting, and am overall pretty happy with it. I opted to have it test up to our current practices (including the mistake `main` "queues" that don't do anything), leaving room to tighten things up in the near future, but at least documenting the current state.